n4js-libs/packages/n4js-runtime/src/js/rt/N4RuntimeBootstrap.js

```diff
@@ -64,7 +52,7 @@
      *                                interfaces defined in definition files without annotation @N4JS.
      * @param instanceMethods - An object holding the methods for the class instance and mixed in methods
      * @param staticMethods - An object holding the descriptors for the class static methods
-     * @param n4typeFn - The factory function to create the meta type.
+     * @param n4typeFn - Optional factory function to create the meta type.
```

Comment:
As we discussed yesterday, we cannot make reflection optional before we had a look at the classes (i.e. trying to use ES6 classes). Making this parameter optional right now will make it more difficult to understand what's going on when looking at the classes without having a true benefit right now. Or am I overlooking something?

Reply:
No, this was just a prework to be able to omit reflection in the future.